src/core/unit.c

```diff
@@ -1336,6 +1337,10 @@ int unit_load(Unit *u) {
                         goto fail;
                 }
 
+                if (u->job_running_timeout != USEC_INFINITY && u->job_running_timeout > u->job_timeout) {
+                        log_unit_warning(u, "JobRunningTimeout is greater than JobTimeout, the latter will override.");
+                }
```

Comment:
also, in all documentation and log messages we suffix config file settings with "=" to indicate that they are config file settings. hence, please write this as "JobRunningTimeout= is greater than JobTimeout="...